CHANGELOG.next.asciidoc

```diff
@@ -170,6 +170,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fixed data type for isr field in `kafka/partition` metricset {pull}10307[10307]
 - Fixed data types for various hosts fields in `mongodb/replstatus` metricset {pull}10307[10307]
 - Added function to close sql database connection. {pull}10355[10355]
+- Fix flaky test of the transaction field on 'transaction_log' Metricset from MSSQL Metricbeat module {pull}10480[10480]
```

Comment:
@sayden No need to add this to the changelog. We normally only add things to the changelog which affect the user.

Reply:
Reverted